Gemfile

```
@@ -0,0 +1,56 @@
+gem "devise", "~> 4.9"
+gem "rspec-rails", "~> 7.1"
+gem "factory_bot_rails", "~> 6.4"
+gem "capybara", "~> 3.40"
+gem "database_cleaner", "~> 2.1"
+gem "shoulda-matchers", "~> 6.5"
+
+gem "tailwindcss-rails", "~> 4.2"
+gem "turbo-rails", "~> 2.0"
+gem "stimulus-rails", "~> 1.3"
+
+group :development, :test do
```

RuboCop (1.75.5)

[warning] Gem group `:development, :test` already defined on line 26 of the Gemfile.

(Bundler/DuplicatedGroup)